src/transpiler.js

```diff
@@ -1,3 +1,5 @@
+const padStart = require('lodash/padStart')
```

Comment:
we don't have lodash has part of our dep. Can you add it?

Reply:
Good catch, we use it everywhere but indeed it was not here. Fixed!